.devcontainer/bin/postCreateCommand.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -e
+
+export BUN_VERSION="v1.1.29"
+export NODE_VERSION="20"
+export NVM_VERSION="v0.40.0"
+export PNPM_VERSION="9.11.0"
+
+apt update
+apt upgrade -y
+
+apt install -y git curl unzip vim
+
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/$NVM_VERSION/install.sh | bash
+
+export NVM_DIR="$HOME/.nvm"
+[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
+[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+
+nvm install "$NODE_VERSION"
```

Comment:
To simplify the setup, how about using [NodeSource's binary distribution][1] instead of nvm? It's just an idea.

[1]: https://github.com/nodesource/distributions?tab=readme-ov-file#debian-and-ubuntu-based-distributions

Reply:
I usually use `nvm` because it can manage many versions of Node.js. For example, we can check if some code works on 18, 20 and 22 too in local environment.

And the reason why I used nvm to install NodeJS, it is recommended way by Node.js website

![image](https://github.com/user-attachments/assets/83a0d693-ddca-4d5d-8945-ef56c67995d7)

I don't disagree with your opinion, just I don't know the advantage of NodeSource.

Reply:
Okay, fair enough!